packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -263,8 +263,8 @@ const createEntityPageListItems = (
         </td>
         <td>
           ${
-            accountId
-              ? `<a href="${formatContentLink(uid)}"> ${entityId} </a>`
+            accountId && universe
+              ? `<a href=${formatContentLink(uid)}> ${entityId} </a>`
               : `${entityId}`
```

Comment:
Just use entityId. No need to string interpolate.

Reply:
Oh I see